src/components/v1/V1Project/ProjectActivity/eventElems/ReservesEventElem.tsx

```diff
@@ -77,15 +68,13 @@ export default function ReservesEventElem({
 
         <div style={{ textAlign: 'right' }}>
           <div style={smallHeaderStyle(colors)}>
-            {printReservesEvent.timestamp && (
-              <span>
-                {formatHistoricalDate(printReservesEvent.timestamp * 1000)}
-              </span>
+            {event.timestamp && (
+              <span>{formatHistoricalDate(event.timestamp * 1000)}</span>
             )}{' '}
-            <EtherscanLink value={printReservesEvent.txHash} type="tx" />
+            <EtherscanLink value={event.txHash} type="tx" />
           </div>
           <div style={smallHeaderStyle(colors)}>
-            called by <FormattedAddress address={printReservesEvent.caller} />
+            <Trans>called by</Trans> <FormattedAddress address={event.caller} />
```

Comment:
**praise** noice, just smol suggestion to include the value in the translation

```suggestion
            <Trans>called by <FormattedAddress address={event.caller} /></Trans>
```

Reply:
why is that? seems like there wouldn't be a value to translate—it's either an address or ENS name anyway? maybe i'm missing something abt translations